test/plugins/features/movies/controller.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie      = require('../../../../lib/models/movie');
+
+describe('movie controller', () => {
+
+  describe('create', () => {
+
+    it('creates a movie', () => {
+      const payload = { title: 'Dom 5ever' };
+
+      return Controller.create(payload)
+      .then((movie) => {
+        expect(movie.get('title')).to.eql(payload.title);
+
+        return new Movie({ id: movie.id }).fetch();
+      })
+      .then((movie) => expect(movie.get('title')).to.eql(payload.title));
```

Comment:
It's up to you. I generally tend to not short-hand expects but there's usually more than just 1. I think I'm used to seeing them in a block so I don't use short-hand but it's up to you.